default/scripting/buildings/CULTURE_LIBRARY.focs.py

```diff
@@ -0,0 +1,52 @@
+from buildings.buildings import SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS
+from focs._effects import (
+    ContainedBy,
+    Destroy,
+    EffectsGroup,
+    Happiness,
+    IsSource,
+    NamedReal,
+    Object,
+    Planet,
+    Population,
+    SetTargetResearch,
+    Source,
+    Value,
+)
+from macros.base_prod import BUILDING_COST_MULTIPLIER
+from macros.priorities import TARGET_AFTER_SCALING_PRIORITY
+
+try:
+    from focs._buildings import *
+except ModuleNotFoundError:
+    pass
+
+BuildingType(  # type: ignore[reportUnboundVariable]
+    name="BLD_CULTURE_LIBRARY",
+    description="BLD_CULTURE_LIBRARY_DESC",
+    buildcost=200 * BUILDING_COST_MULTIPLIER,
+    buildtime=1,
+    tags="ANTIQUATED",
```

Comment:
default/scripting/buildings/CULTURE_LIBRARY.focs.py:29: error: Argument "tags" to "BuildingType" has incompatible type "str"; expected "list[str] | None"  [arg-type]

Reply:
I forgot to run a check for all files after I changed typing.